app/frontend/actions.js

```diff
@@ -251,9 +292,9 @@ module.exports = ({setState, getState}) => {
 
   const openAddressDetail = (state, {address, bip32path}, copyOnClick) => {
     /*
-    * because we don't want to trigger trezor address
-    * verification for the  donation address
-    */
+     * because we don't want to trigger trezor address
```

Comment:
let's extract this into separate commit which we should add on top of both master and develop